docs/content/scripts/analytics/google-analytics.md

```diff
@@ -27,7 +27,8 @@ export default defineNuxtConfig({
   scripts: {
     registry: {
       googleAnalytics: {
-        id: 'YOUR_ID'
+        id: 'YOUR_ID',
+        dataLayerName: 'defaultGa'
```

Comment:
end user should not need to define this, it should only be listed under docs for adding multiple google analytics